app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const { getTopics } = require("./controllers/app.controllers");
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.get("/api/*", (req, res) => {
+  res.status(404).send({ msg: "Invalid URL Passed" });
```

Comment:
Consider to move you error handling function to a separate files as well

Reply:
Created new error handling folder/file and exported